.eslintignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 coverage
-
+dist
```

Comment:
Also add the dist folder to the .gitignore file and explain steps required to generate it.